### PR TITLE
Add a historical feature for health stats

### PR DESCRIPTION
**Pull request by thecoop**
Add a historical feature for support of the health stats action

### CODE REVIEW

File: server/src/main/java/org/elasticsearch/health/HealthFeatures.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 public class HealthFeatures implements FeatureSpecification {
 
-    public static final NodeFeature SUPPORTS_HEALTH = new NodeFeature("supports_health");
+    public static final NodeFeature SUPPORTS_HEALTH = new NodeFeature("health.supports_health");
```
Comment (thecoop) [inline server/src/main/java/org/elasticsearch/health/HealthFeatures.java:19]: Historical features are not actually published anywhere, so it's ok to change the id here

**Comment by thecoop** [comment]
@elasticmachine rerun elasticsearch-ci

**Comment by thecoop** [comment]
@elasticsearchmachine rerun elasticsearch-ci

**Comment by thecoop** [comment]
@elasticmachine update branch

**Comment by thecoop** [comment]
@elasticmachine update branch

**Comment by thecoop** [comment]
@elasticmachine update branch

File: server/src/main/java/org/elasticsearch/health/HealthFeatures.java (hunk 1 above)
Comment (gmarouli) [reply server/src/main/java/org/elasticsearch/health/HealthFeatures.java:19]: Ah, thanks, TIL.

**Review by gmarouli — APPROVED** [review]
@thecoop thank you for updating this. LGTM!

**Pull request merged**